tools/js-sdk-release-tools/src/test/changelog/changelogGenerator.test.ts

```diff
@@ -5,8 +5,33 @@ import { SDKType } from "../../common/types.js";
 import { describe } from "node:test";
 import { tryReadNpmPackageChangelog } from "../../common/utils.js";
 import { removeSync, outputFileSync } from "fs-extra";
+import { getFirstReleaseContent } from "../../common/changelog/modifyChangelogFileAndBumpVersion.js";
 
 describe("Breaking change detection", () => {
+    test("ModularClient ->firstGA", async () => {
+        const root = path.join(__dirname, 'testCases/modular-firstRelease/');
+        const content = getFirstReleaseContent(root,true);
```

Comment:
```suggestion
        const content = getFirstReleaseContent(root, true);
```
please make sure the format in this file